artman/tasks/gapic_tasks.py

```diff
@@ -66,14 +66,15 @@ def _move_to(self, gapic_config_dir, gapic_api_yaml):
         conf_out = os.path.abspath(gapic_api_yaml[0])
         if os.path.exists(conf_out):
             # TODO (issue #80): no need to test in remote environment
-            raise ValueError(error_fmt + 'File already exists')
-        else:
-            return conf_out
+            olderVersion = conf_out + '.old'
+            print('File already exists, save the old version as ' + olderVersion)
+            self.exec_command(['mv', conf_out, olderVersion])
```

Comment:
This will fail if running twice. I won't bother to make a perfect change history though.